### PR TITLE
Enhancement/1503 e2e use proxy auth

### PR DESCRIPTION
**Pull request by ryanwelcher**
## Summary

Addresses issue #1503 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: tests/e2e/specs/admin-tracking.test.js [hunk 1]
```diff
@@ -16,16 +16,16 @@ async function toggleOptIn() {
...
+		await activatePlugin( 'e2e-tests-proxy-auth-plugin' );
+		await activatePlugin( 'e2e-tests-site-verification-plugin' );
 		await setSearchConsoleProperty();
 
 		await visitAdminPage( 'admin.php', 'page=googlesitekit-settings' );
-		await page.waitForSelector( '.mdc-tab-bar' );
+		await page.waitForSelector( '.mdc-tab-bar button.mdc-tab' );
+		await expect( page ).toMatchElement( 'button.mdc-tab', { text: 'Admin Settings' } );
+
+		await page.waitFor( 80 ); // I am not sure what but we need this delay before clicking on the button.
```
[the remark this answers is not in the document]
Comment (ryanwelcher) [reply tests/e2e/specs/admin-tracking.test.js:28]: @aaemnnosttv wondering if we should make this a constant? `E2E_PAGE_DELAY`

[the remark this answers is not in the document]
Comment (aaemnnosttv) [reply tests/e2e/specs/admin-tracking.test.js]: Yeah I had a similar thought. Either that or maybe a utility? Or both? 😄

[the remark this answers is not in the document]
Comment (ryanwelcher) [reply tests/e2e/specs/admin-tracking.test.js]: A utility that can take a delay parameter but defaults to that constant could work. We could export both?

**Comment by aaemnnosttv** [comment]
@googlebot I consent.

File: tests/e2e/specs/modules/analytics/amp-validation.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Internal dependencies
+ */
+import {
+	activateAMPWithMode,
+	setupAnalytics,
+	setupSiteKit,
+	resetSiteKit,
+} from '../../../utils';
+
+describe( 'AMP homepage validates with analytics enabled', () => {
```
Comment (aaemnnosttv) [inline tests/e2e/specs/modules/analytics/amp-validation.test.js:34]:
```suggestion
describe( 'AMP homepage validates with Analytics enabled', () => {
```

**Pull request merged**